requirements.tests.txt

```diff
@@ -4,5 +4,9 @@ pytest-cov>=2.3.1
 pytest-postgresql>=1.3.0
 pytest-xdist>=1.22.0
 setuptools>=35.0.0
-eventlet
-gevent
+
+# for integration tests
+eventlet==0.22.0
+gevent==1.2.2
+redis==2.10.6
+SQLAlchemy==1.1.14
```

Comment:
is the strict version selector deliberate? What if downstream projects use different versions?

Reply:
This is for tests, so downstream won't use it, and is fixed in an upcoming PR anyway

Reply:
Fixed as it just specifies a minimum